pkg/provisioner/ironic/hardwaredetails/hardwaredetails.go

```diff
@@ -24,6 +28,21 @@ func GetHardwareDetails(data *introspection.Data) *metal3v1alpha1.HardwareDetail
 	return details
 }
 
+func getLLDPProcessed(intf introspection.BaseInterfaceType) metal3v1alpha1.LLDPProcessed {
+	lldpjson, err := json.Marshal(intf.LLDPProcessed)
```

Comment:
I hope there is a more efficient way to get the structure than to serialize it and deserialize back...

Reply:
@dtantsur 
i can't think of a good way.
do you have any code to suggest?

Reply:
instead of dump the entiry data, how about saving only specific values ​​first?